### PR TITLE
Package morbig.0.9

### PR DESCRIPTION
**Pull request by Niols**
### `morbig.0.9`
A trustworthy parser for POSIX shell
Morbig is a parser for shell scripts written in the POSIX shell script
language. It parses the scripts statically, that is without executing
them, and constructs a concrete syntax tree for each of them.  The
concrete syntax trees are built using constructors according to the
shell grammar of the POSIX standard.

---
* Homepage: https://github.com/colis-anr/morbig
* Source repo: git://github.com/colis-anr/morbig.git
* Bug tracker: https://github.com/colis-anr/morbig/issues

---
:camel: Pull-request generated by opam-publish v2.0.0~beta

### CODE REVIEW

**Comment by mseri** [comment]
Sorry again, the package format is opam 2.0.0, but the PR should be still targeting opam 1.2 (there is an automatic process that will convert the merged files for the opam 2.0.0 branch)

**Comment by Niols** [comment]
Oh, you don't have to be sorry, I feel like I am wasting your time here. I should never have gone for opam-publish 2.0 ;-)

Is it better now?

File: packages/morbig/morbig.0.9/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+]
+license: "GPL3"
+
+homepage: "https://github.com/colis-anr/morbig"
+bug-reports: "https://github.com/colis-anr/morbig/issues"
+dev-repo: "git://github.com/colis-anr/morbig.git"
+
+available: [os != "macos"]
+depends: [
+  "ocaml"                {build & >= "4.03.0"}
```
Comment (mseri) [inline packages/morbig/morbig.0.9/opam]: This does not work in opam 1.2, you need to modify availabe as follows:
```
available: [os != "macos" & ocaml-version >= "4.03.0"]
```
or something like that. I can't remember if opam 1.2 wants "osx", "darwin" or "macos", sorry.

**Comment by Niols** [comment]
How come my PR still passes the tests for 1.2 by the way?

Anyway, I've looked in the repo, it seems to be "macos" and I've made the right changes.

What do you think of the url file? Is it OK to have the two checksums or is it also 2.0 only?

**Comment by mseri** [comment]
I thought the multiple checksums where fine, but after looking at the documentation I think they are a thing of opam2 as well. Thanks for spotting it.

The CI is already using opam2-based images, so I think it may be a bit weak with respect to that. Probably camelus will complain.

I really can't wait to move to opam2 to be honest :)

**Comment by Niols** [comment]
Uh. Turns out camelus complains about opam-1.2 features :-o
Also, when I take a look at the files in the repository, they seem to have been updated to use the 2.0 scheme. Am I really supposed to send a PR using the 1.2 scheme?

**Comment by Niols** [comment]
Maybe we don't have to wait for OPAM 2.0 anymore. Maybe it's already here, it the middle of us. Maybe we have been blind for years! :-o

*(can't wait for it either)*

**Comment by mseri** [comment]
The problem is that your PR is not against `master` but against the `2.0.0` branch. You need to point it to the other branch

**Comment by Niols** [comment]
Indeed, thank you for the remark, I'll change all that.

**Comment by Niols** [comment]
I tried to use GitHub's "change branch" mechanism, but it appeared it confused the CI that tries to build the whole world. I'll guess I'll start again from scratch (we'll manage, eventually!)

**Comment by Niols** [comment]
Here it is: #12579. Sorry for all the trouble.

**Pull request closed without merging**